portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,21 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body class="site-background">
+    <div class="header">
+      <h1>Hi, I'm Shreya.</h1>
     </div>
+    <ul>
+        <li><a href="index.html">Home</a></li>
```

Comment:
i think there's some extra indentation here (should only be 2 between ul and li)